### DISCUSSION TITLE
FileMaker e API 2.0

### DISCUSSION THREAD

**Discussion by DemetrioXY** [fattureincloud/api#279] [Q&A] [answered]
I'm a FileMaker developer and a complete newbie to the new FattureInCloud API 2.0.

Where can I find a simple application or samples in FileMaker to start from scratch?

With basic info like:
- how to set the connection, credentials, parameters
- data format, JSON samples
- what samples to use on the documentation to start, what are the nearest to the real application in FileMaker

Thank you!
Demetrio

-----------------------------------------------------------------------------------------------------------

Sono uno sviluppatore FileMaker e completo newbie per quanto riguarda le nuove API 2.0 di FattureInCloud.

Dove posso trovare una semplice applicazione o esempi in FileMaker per iniziare da zero?

Con informazioni di base, tipo:
- come impostare il collegamento, le credenziali, i parametri
- formato dei dati ed esempi in JSON
- quali esempi usare sulla documentazione per iniziare, quali sono più vicini alla reale applicazione in FileMaker

Grazie!

Demetrio

**Comment by emajo** [comment]
Ciao @DemetrioXY, purtroppo non conosciamo FileMaker, le consiglierei di scrivere su [questa discussione](https://github.com/fattureincloud/api/discussions/88) dove ci sono sviluppatori che hanno lavorato a integrazioni simili.
Per quanto riguarda invece gli esempi json, ne abbiamo per ogni chiamata sulla nostra [API Reference](https://developers.fattureincloud.it/api-reference) oltre a spiegazioni più approfondite delle chiamate più importanti documentate sul nostro [portale developers](https://developers.fattureincloud.it/docs/guides/).

Grazie,
Emanuele.

**Reply by DemetrioXY** [reply]
Grazie! Proverò nell'altra discussione, a meno che qualcuno non posti qui delle informazioni.

**Reply by valmoz** [reply]
Salve @DemetrioXY 
penso che potranno esserle utile anche le informazioni disponibili a [questa pagina](https://developers.fattureincloud.it/docs/quickstarts/vanilla-code)

Non conosciamo FileMaker, ma da quanto ho capito utilizza codice C/C++ che potrebbe essere generato come indicato in pagina. Spero di non sbagliarmi a riguardo.

Nel caso, può comunque postare i suoi dubbi in questa discussion, proveremo ad aiutarla per quanto possibile.

Grazie
Mauro

**Reply by DemetrioXY** [reply]
Grazie mille!

**Reply by valmoz** [reply]
Ciao @DemetrioXY possiamo chiudere questa discussione?

**Reply by DemetrioXY** [reply]
Salve, @valmoz. Certamente, visto che non ci sono altri contributi.
Spero di ricevere qualche risposta nell'altra discussione.
Grazie

**Comment by valmoz** [comment, marked as answer]
le invio qualche guida che forse potrebbe esserle utile.

- https://isolutions.medium.com/integrating-apis-into-filemaker-apps-85a821c814
- https://www.productivecomputing.com/blog/api-integration-with-filemaker/
- https://help.claris.com/it/data-api-guide/content/index.html
- https://www.youtube.com/watch?v=aVwP84QLtdE
- https://dbservices.com/blog/integrating-filemaker-with-rest-apis
- https://www.youtube.com/watch?v=lsMQRaeKNDk&t=3s
- https://developers.fattureincloud.it/docs/general-knowledge/rest-apis

**Reply by DemetrioXY** [reply]
Perfetto, grazie!
Ottimi riferimenti.
